composer/datasets/brats.py

```diff
@@ -264,8 +264,8 @@ def get_split(data, idx):
 def get_data_split(path: str):
     try:
         from sklearn.model_selection import KFold
-    except ImportError as e:
-        raise ImportError(
+    except MissingConditionalImportError as e:
+        raise MissingConditionalImportError(
             textwrap.dedent("""\
```

Comment:
The `MissingConditionalImportError` signature takes three arguments instead of an error string. Please see https://github.com/mosaicml/composer/blob/d765fa23909dc1f14f27fecde194d3e4842670d3/composer/utils/import_helpers.py#L12.